api/src/utils/helpers.ts

```diff
@@ -4,11 +4,12 @@ export interface SessionUser {
   firstName: string;
   lastName: string;
   preferredName: string;
+  profilePic: string;
   email: string;
   phone: string;
   gender: [string];
   pronouns: [string];
-  dateJoined: Date;
+  dateJoined: string;
```

Comment:
Why did we change this to a string?

Reply:
Sorry, screenshot was old

Reply:
also type script was bullying me that's why

Reply:
actually nvmd, TS WASSS bullying me and then I bullied it back to work. I just forgot to change some stuff back